app/components/supplier_table_component.html.haml

```diff
@@ -8,6 +8,12 @@
         %th{ scope: "col" }
           .cads-table__content
             Supplier
+        %th{ scope: "col" }
```

Comment:
I think in the updated designs the table heading reads `Overall rating` on both mobile and desktop

Reply:
The ticket is specifically to address the zoom - text content is handled separately by a different team. The design is quite old, and the content on the design may not reflect the current intentions of the content team. I will double check with them, however.

Reply:
Now changed